session2/chapter2/lightning-internal.md

```diff
@@ -1,147 +1,155 @@
-# TiDB Lightning 简介
-TiDB Lightning 是一个将全量数据高速导入到 TiDB 集群的工具，速度可达到传统执行 SQL 导入方式的至少 3 倍、大约每小时 100 GB，有以下两个主要的使用场景：一是大量新数据的快速导入；二是全量数据的备份恢复。目前，支持 Mydumper 或 CSV 输出格式的数据源。你可以在以下两种场景下使用 Lightning：
+# 2.2.1 TiDB Lightning 简介
+TiDB Lightning 是一个将全量数据高速导入到 TiDB 集群的工具，速度可达到传统执行 SQL 导入方式的 3 倍以上、大约每小时 100 GB。Lightning 有以下两个主要的使用场景：一是大量新数据的快速导入、二是全量数据的备份恢复。目前，Lightning 支持 Mydumper 或 CSV 输出格式的数据源。

```

Comment:
100 GB or 1000 GB ?

Reply:
Our documentation actually said 300 GB/h. 

1000 GB/h is the maximum possible speed with one importer and 3 replicas without considering any cost of computation, which is not really possible with a single Lightning/Importer pair.